tests/define_test_aggregate_project.py

```diff
@@ -0,0 +1,77 @@
+from flow import FlowProject, aggregator, cmd, with_job
+
+
+class _TestAggregateProject(FlowProject):
+    pass
+
+
+def custom_aggregator(jobs):
+    even = [job for job in jobs if job.sp.i % 2 == 0]
+    odd = [job for job in jobs if job.sp.i % 2 != 0]
+    return [even, odd]
+
+
+group1 = _TestAggregateProject.make_group(name="group_agg", aggregator_obj=aggregator())
+
+
+@_TestAggregateProject.operation
+def op1(job):
+    pass
+
+
+@_TestAggregateProject.operation
+@aggregator.groupby("even")
+def agg_op1(*jobs):
+    total = sum([job.sp.i for job in jobs])
+    for job in jobs:
+        job.document.sum = total
```

Comment:
All your tests are doing this. I'd define a small function
```
def set_all_job_docs(jobs, key, val):
    for job in jobs:
        job[key] = val
```
and use that. Then half of these operations become trivial (setting `True`) and the others do one calculation and call this.

Reply:
Done in 6266b3728188381fc4af60dadd9fc9f074f9dab1.